src/Components/InputTodo.js

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { runToast } from './toast/Toast';
+
+const InputTodo = ({ list, setList }) => {
+    const [form, setForm] = useState('');
+    const handleChange = (e) => {
+        setForm(e.target.value);
+    };
+    const handleInput = () => {
```

Comment:
여기도 이런 이름이 더 좋을 것 같아요

```suggestion
    const handleInputButtonClick = () => {
```